cpp/src/cider-velox/src/CiderPlanNode.h

```diff
@@ -35,7 +39,11 @@ class CiderPlanNode : public core::PlanNode {
                          const core::PlanNodePtr& source,
                          const RowTypePtr& outputType,
                          const ::substrait::Plan& plan)
-      : core::PlanNode(id), sources_({source}), plan_(plan), outputType_(outputType) {}
+      : core::PlanNode(id), sources_({source}), plan_(plan), outputType_(outputType) {
+    auto translator = generator::SubstraitToRelAlgExecutionUnit(plan);
+    RelAlgExecutionUnit ra_exe_unit = translator.createRelAlgExecutionUnit();
+    codegen_context_ = cider::exec::nextgen::compile(ra_exe_unit);
```

Comment:
Why call it `codegen_context_`? maybe `compile_res` better?

Reply:
it's just moved from another place. and the structure is `CodegenContext`